### PR TITLE
update hugo to 0.120.4

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/istio/istio.io/issues/14094
Let me know when the thread can be updated branch.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     publish = "public"
 
 [build.environment]
-    HUGO_VERSION = "0.119.0"
+    HUGO_VERSION = "0.120.3"
```
Comment (craigbox) [inline netlify.toml]:
```suggestion
    HUGO_VERSION = "0.120.4"
```

**Review by craigbox — CHANGES_REQUESTED** [review]

File: netlify.toml (hunk 1 above)
Comment (Arhell) [reply netlify.toml]: Thank you, already

**Comment by ericvn** [comment]
Just adding the DNM label to be removed once we have branched.

**Review by ericvn — APPROVED** [review]
LGTM, and we have branched.

**Comment by craigbox** [comment]
LGTM!

**Pull request merged**